.devops/deploy-pipelines.yml

```diff
@@ -0,0 +1,152 @@
+          )
+        )
+      )
+    pool:
+      vmImage: 'ubuntu-latest'
+    jobs:
+      - job: make_release
+        steps:
+        - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/master') }}:
+          - template: azure-templates/make-release-steps.yml
+            parameters:
+              semver: '${{ parameters.RELEASE_SEMVER }}'
+              gitEmail: $(GIT_EMAIL)
+              gitUsername: $(GIT_USERNAME)
+              gitHubConnection: $(GITHUB_CONNECTION)
+        - ${{ if ne(variables['Build.SourceBranch'], 'refs/heads/master') }}:
+          - script: |
+              echo "We assume this reference to be a valid release: $(Build.SourceBranch). Therefore, there is no need to bundle a new release."
+            displayName: 'Skip release bundle'
+
+
+  # Prepare Artifact
+  - stage: Prepare_artifact
+    jobs:
+     - job: 'make_artifact'
+        steps:
+          # Build application
+          - template: ./make-build-steps.yml
+            parameters:
+              make: predeploy_build
+          # Install functions extensions
+          - task: DotNetCoreCLI@2
+            inputs:
+              command: "build"
+              arguments: "-o bin"   
+          # Copy application to  
+          - task: CopyFiles@2
+            inputs:
+              SourceFolder: '$(System.DefaultWorkingDirectory)'
+              TargetFolder: '$(Build.ArtifactStagingDirectory)'
+              Contents: |
+                **/*
+                !.git/**/*
+                !**/*.js.map
+                !**/*.ts
+                !.vscode/**/*
+                !azure-templates/**/*
+                !azure-pipelines.yml
+                !.prettierrc
+                !.gitignore
+                !README.md
+                !jest.config.js
+                !local.settings.json
+                !test
+                !tsconfig.json
+            displayName: 'Copy deploy files'
+
+  # Deploy to PRODUCTION environment
+  - stage: Deploy_production
+    dependsOn:
+      - Release
+    jobs:
+        - job: 'do_deploy'         
+           # First step: deploy to 'staging' slot 
+          - task: AzureFunctionApp@1  
+            inputs:
+              azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
+              resourceGroupName: '$(PRODUCTION_RESOURCE_GROUP_NAME)'
+              appType: 'functionApp'
+              appName: '$(PRODUCTION_APP_NAME)'
+              package: '$(Build.ArtifactStagingDirectory)/'
+              deploymentMethod: 'auto'
+              deployToSlotOrASE: true
+              slotName: 'staging'
+            displayName: Deploy to staging slot
+           # Second step: check the app is working
+          - template: templates/rest-healthcheck/template.yaml@pagopaCommons 
+            parameters:
+              azureSubscription: '$(PRODUCTION_AZURE_SUBSCRIPTION)'
+              appName: '$(PRODUCTION_APP_NAME)'
+              endpoint: 'https://$(PRODUCTION_APP_NAME)-staging.azurewebsites.net/info'
```

Comment:
isn't /api/v1/info? 
isn't it better to take only the path (/api/v1/info) from an env? or take the path + the URL with
two default values (https://$(PRODUCTION_APP_NAME)-staging.azurewebsites.net for the URL, /info for the path)

Reply:
I'd take the whole endpoint as it isn't so much an effort to specify, but it may be hard to abstract for other cases

Reply:
The correct path should be `/api/info`

Reply:
> I'd take the whole endpoint as it isn't so much an effort to specify, but it may be hard to abstract for other cases

is this the only parameter that differs between functions azure-template.yml? (it looks like everything else is the same for every function)